include/jemalloc/internal/decay.h

```diff
@@ -146,4 +146,14 @@ void decay_reinit(decay_t *decay, nstime_t *cur_time, ssize_t decay_ms);
 bool decay_maybe_advance_epoch(decay_t *decay, nstime_t *new_time,
     size_t current_npages);
 
+typedef struct purge_interval_opts_s purge_interval_opts_t;
```

Comment:
This is the way we'd ordinarily write an options struct, although in this case I think we can pare down the details enough that we don't need one here.

(We also generally tend to reserve options structs for things like configuration settings rather than function arguments, although that's more of a case-by-case sort of thing).